src/Test/TestCombat.java

```diff
@@ -43,7 +46,8 @@ private boolean testUnitDeath()
     Unit infB = addUnit(testMap, testCo2, UnitEnum.INFANTRY, 1, 2);
 
     // Make sure the infantry will die with one attack
-    infB.damageHP(7);
+    CombatDamageModifier mod = new CombatDamageModifier(testCo1,500);
```

Comment:
That's... quite a power boost. Do we still want a vanilla test case for combat? I might be more comfortable making a separate test for the combat modifier. Or even make a proper unit test for the CombatModifier class.